requirements.txt

```diff
@@ -1,4 +1,4 @@
-IBMQuantumExperience>=1.9.8
+IBMQuantumExperience>=2.0.1
```

Comment:
Thanks for remembering to bump the dependency and avoid an extra PR just for that :tada: